tests/unit/test_shorthand.py

```diff
@@ -184,6 +202,63 @@ def test_parse(data, expected):
     actual = shorthand.ShorthandParser().parse(data)
     assert actual == expected
 
+class TestShorthandParser:
+    @pytest.fixture()
+    def files(self):
+        files = FileCreator()
+        yield files
+        files.remove_all()
+
+    def test_paramfile(self, files):
```

Comment:
I noticed a lot of these tests have the same structure. Let's use `pytest.mark.parametrize` to clean it up a bit. You can see an [example here](https://github.com/aws/aws-cli/blob/88e1f9c6697e118652787caf3194c6464c9c8068/tests/unit/test_shorthand.py#L197).